src/test/java/com/hellosign/sdk/HelloSignClientTest.java

```diff
@@ -637,6 +637,18 @@ public void testGetFilesInvalid() throws Exception {
         client.getFiles("foo");
     }
 
+    @Test(expected = HelloSignException.class)
+    public void testGetFilesUrlInvalid() throws Exception {
+        mockResponseCode(404);
+        client.getFilesUrl("foo");
```

Comment:
Nit style thing, but prefer to use more real world values for such things as the signature id in this test